docusaurus/website/docusaurus.config.base.js

```diff
@@ -113,6 +113,14 @@ const plugins = [
           from: ['/ui-extensions/create-an-extension-point'],
           to: '/how-to-guides/ui-extensions/create-an-extension-point',
         },
+        {
+          from: ['/create-a-plugin/extend-a-plugin/include-dashboards'],
+          to: '/how-to-guides/app-plugins/include-dashboards-in-app-plugins',
```

Comment:
I'd suggest removing this redirect
- its a fairly new topic so less impact
- we're splitting the content into multiple locations
- data sources are more common than apps, so the not-yet-created page would be a more appropriate redirect **IF** we need one